### PR TITLE
feat: always focus prompt

### LINKED ISSUE #1856: Focus the prompt
### Feature description

Sometimes it happens that the prompt looses focus and is hidden behind the actual browser window.

Ideally the prompt is always on top - or gets focus back after some time. 

### Describe the solution

Seems we can do `browser.window.update()` with `focused` and/or `drawAttention` in [here](https://github.com/getAlby/lightning-browser-extension/blob/master/src/common/lib/utils.ts#L77). Maybe with a setInterval?

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/update

### Describe alternatives

_No response_

### Additional context

_No response_

### Are you working on this?

None

### PR DESCRIPTION
**Pull request by bumi**
### Describe the changes you have made in this PR

This makes sure that the prompt is focussed every 2 seconds. 
If the user "looses" the prompt this change brings the prompt in focus again.

### Link this PR to an issue [optional]

Fixes #1856

### Type of change

- `feat`: New feature (non-breaking change which adds functionality)

### Video

https://share.cleanshot.com/ZmEgc6

### CODE REVIEW

**Review by reneaaron — APPROVED** [review]
tACK

File: src/common/lib/utils.ts [hunk 1]
```diff
@@ -45,7 +46,7 @@ const utils = {
   openUrl: (url: string) => {
     browser.tabs.create({ url });
   },
-  openPrompt: <Type>(message: {
+  openPrompt: async <Type>(message: {
```
Comment (bumi) [inline src/common/lib/utils.ts:49]: does this need to be async? if this is async I think then line 75 should not `return new Promise`

Comment (reneaaron) [reply src/common/lib/utils.ts:49]: We need to await the async function getWindowPosition, therefore it has to be async.

https://github.com/getAlby/lightning-browser-extension/pull/1883/files#diff-4bc1a33d21997d321e25376ba8b1376f5a25afc2b6fb9450515547c9aec95d66R73

Comment (bumi) [reply src/common/lib/utils.ts:49]:
```
return getWindowPosition(windowWidth, windowHeight).then(dimensions => {

})
```

**Pull request merged**